myriadeploy/get_logs.py

```diff
@@ -28,30 +27,29 @@ def get_error_logs_from_worker(hostname, dirname, username,
                                worker_id, description):
     mkdir_if_not_exists(description)
     if hostname == 'localhost':
-        uri = "%s/worker_%s_stderr" % (dirname, worker_id)
+        uri = "%s/worker_%s/worker_%s_stderr" % (dirname, worker_id, worker_id)
```

Comment:
Why do we need to qualify the name of files under a worker-specific directory with the worker ID?